README.txt

```diff
@@ -15,7 +16,7 @@ fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut,
 imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium.
 Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate
 eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac,
-enim. Mauqila lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus
+enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus
 viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean imperdiet. Etiam
```

Comment:
range started on non-shifted side on modified line and ending on shifted side on non-modified line